practice_problems_2/passByRef/main.php

```diff
@@ -0,0 +1,11 @@
+<?php
+function double_val (&$input) {
```

Comment:
There should not be space after function name.

Reply:
Done